doc/index.rst

```diff
@@ -5,18 +5,17 @@ Welcome to SpikeInterface's documentation!
 
 .. warning::
 
-    This is the not yet released documentation of spikeinterface (0.90.0)
+    This is the documentation for SpikeInterface (0.90.0)
     
-    To see reelased documention this is here https://spikeinterface.readthedocs.io/en/stable
-    
-    Actual  API released is 0.12.0.
+    To see the documentation for the old API (0.1X) see: https://spikeinterface.readthedocs.io/en/stable
```

Comment:
This will not be true very soon put the exact URL for old API

Reply:
we change it when we release